protocol/messenger_handler.go

```diff
@@ -1074,9 +1074,15 @@ func (m *Messenger) HandleChatMessage(state *ReceivedMessageState) error {
 		return err // matchChatEntity returns a descriptive error message
 	}
 
-	if chat.ReadMessagesAtClockValue > state.CurrentMessageState.WhisperTimestamp {
-		receivedMessage.Seen = true
-	}
+	// This will prevent older messages to be marked as seen, currently the UI/UX behavior is unspecified,
```

Comment:
Perhaps add a `TODO` to flag that this needs to be address in the future?

Reply:
it didn't work, @Samyoul pls review again, thank you